dom.bs

```diff
@@ -499,6 +505,15 @@ list of tuples, each of which consists of an <b>item</b> (an {{EventTarget}} obj
 object). A tuple is formatted as (<b>item</b>, <b>target</b>, <b>relatedTarget</b>). A <a
 for=Event>path</a> is initially the empty list.</p>
 
+For web compatibility, the <a>current global object</a> has an
```

Comment:
All new/changed paragraphs should start with `<p>`.

This paragraph should be reworded and moved done I think. It makes more sense to me to define this extension to the `Window` object at the end of this section since it's also at the end of the IDL.

Furthermore, we shouldn't give the reason in the description of the feature. That can go into a note if it's important.